src/ts/components/account/NetworkCard.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+interface INetworkCardProps {
+  imgPath: string
+  name: string
+  supported: boolean
+}
+
+interface INetworkCardState {
+  selected: boolean
+}
+
+export default class NetworkCard extends React.Component<INetworkCardProps, INetworkCardState> {
+
+  state = {
+    selected: false
+  }
+
+  render () {
+    return this.state.selected ? this.renderClicked() : this.renderDefault()
+  }
+
+  renderDefault () {
```

Comment:
Is there any diff between `renderDefault` and `renderClicked`? they look same to me

Reply:
Oh so clicking card will make check icon appear in theme color.

Reply:
I've added the effect

<img src="https://i.imgur.com/c6yxI6V.png" width="200"/>

Reply:
Maybe they can be merged with some conditional rendering within one function to reduce code duplication. Anyway it is minor.